woocommerce/woocommerce-gateway-stripe
# refactor: setup needed messaging

# Changes proposed in this Pull Request:

Adjusting the UI of the "Setup needed" messaging from text to a pill

Before:
![Screen Shot 2021-09-15 at 12 04 22 PM](https://user-images.githubusercontent.com/273592/133477769-4911d24e-15f4-4033-bce0-9b851316fbd9.png)


After:
![Screen Shot 2021-09-15 at 12 02 07 PM](https://user-images.githubusercontent.com/273592/133477593-746973d1-ee2a-4d6f-b966-9b1116fda1cd.png)



# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` feature enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- UI should be updated

I didn't add any tests because everything is hardcoded, for now.
